lib/Service/ImportService.php

```diff
@@ -382,30 +380,14 @@ private function createRow(Row $row): void {
 				$hasData = $hasData || !empty($value);
 
 				if ($column->getType() === 'datetime') {
-					if ($column->getType() === 'datetime' && $column->getSubtype() === 'date') {
+					if ($column->getSubtype() === 'date') {
```

Comment:
Could we also add constants for the subtypes like we have for types in https://github.com/nextcloud/tables/blob/c3c97cfbd963edb05ce245241c11f25fc76cdd83/lib/Db/Column.php#L95 ?

I always struggle finding the ones that are used, but may be unrelated

Reply:
added and adjusted this whole service file.